src/System.Drawing.Graphics/src/System/Drawing/Graphics/Image.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace System.Drawing.Graphics
+{
+    public enum PixelFormat
+    {
+        ARGB,
```

Comment:
Do you mean specifically that ARGB should be Argb? The enum PixelFormat is in the Pascal Casing Convention